docs/addons/emr-eks.md

```diff
@@ -0,0 +1,33 @@
+# EMR on EKS Add-on
```

Comment:
Let's reference the team doc here as well, so that customer would know how to approach using this add-on.

Reply:
done